src/components/ClassTile/ClassTile.js

```diff
@@ -0,0 +1,86 @@
+import styled from 'styled-components/macro';
+
+export const ClassTile = ({ data }) => {
+    const timeToString = number => {
+        const string = number.toString();
+        if (string.length === 3) {
+            return string.substring(0, 1) + ':' + string.substring(1, 3);
+        } else {
+            return string.substring(0, 2) + ':' + string.substring(2, 4);
+        }
+    };
+
+    let coachesSet = false;
+    let coachesArray = data.coaches;
+
+    if (coachesArray) {
+        coachesSet = true;
+    }
```

Comment:
You could use the `ternary` operator here:
```suggestion
    const coachesSet = data.coaches ? true : false
```

Read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) about it.